app/services/candidate_interface/pick_course_option.rb

```diff
@@ -0,0 +1,45 @@
+module CandidateInterface
+  class PickCourseOption
```

Comment:
This is great. One advantage is that we can now add more isolated specs for the service business logic. I think we are covered by the system specs for now, service specs can be added when we need them (when we want to change the logic etc.)